Dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG GO_VERSION="1.23"
 ARG ALPINE_VERSION="3.20"
+ARG DISTROLESS_VERSION=debian12@sha256:d71f4b239be2d412017b798a0a401c44c3049a3ca454838473a4c32ed076bfea
```

Comment:
i guess thats a version pin. Do we have a bot that checks for updates?

Reply:
No we do not, yet :D . I can create an issue, thanks